docker/nginx/conf.d/common.conf

```diff
@@ -0,0 +1,12 @@
+include uwsgi_params;
```

Comment:
This is nice! We should check if we can avoid having an extra different config file, to make it easier to deploy this.